src/reducers/starWarsReducer.js

```diff
@@ -1,13 +1,40 @@
-import /* we need our action types here*/ "../actions";
+import /* we need our action types here*/  {FETCHING ,  SUCCESS  ,   FAILURE }       from "../actions";
 const initialState = {
-  characters: []
+  characters: [],
+  fetching:false,
+  error:'',
   // Array characters, Boolean fetching, null error.
```

Comment:
When there is no error, keep it set to `null` instead of an empty string `""` - `null` states more declaratively that there is no error, as opposed to having an 'error' that is composes of empty text.